pkg/products/observability/prometheusRules.go

```diff
@@ -355,7 +355,7 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 						"sop_url": resources.SopApiManagementTenantCRFailed,
 						"message": "An APIManagementTenant CR has failed to reconcile. See the labels for details.",
 					},
-					Expr:   intstr.FromString(`tenants_summary{provisioningStatus!="3scale account ready"}`),
+					Expr:   intstr.FromString(`tenants_summary{provisioningStatus!="3scale account ready"} or tenants_summary{provisioningStatus != "won't provision"}`),
```

Comment:
@Stellar-9 Sorry for the confusion but the conditional should actually be `and` instead of `or`. That's my bad, I gave you the wrong advice in the previous suggestion. This fix should enable the changes to work as expected.
```suggestion
					Expr:   intstr.FromString(`tenants_summary{provisioningStatus!="3scale account ready"} and tenants_summary{provisioningStatus != "won't provision"}`),
```